workflows-scripts/circulation/check-in-check-out/scripts/circ-data-load.sh

```diff
@@ -0,0 +1,59 @@
+# This script needs to be supplied a psql.conf file, which contains the endpoint and credentials of the database.
+# This script needs to be run in the same directory as the loans.tsv, requests.tsv, and patron_action_sessions.tsv"
+# Example usage: $ ./circ-data-load.sh psql.conf fs08000010 
+
+#!/bin/bash -e
+
+if [[ $# -lt 2 ]]; then
+   echo "Usage: $0 <db config> <TENANT> [--clear-loan-audit]";
+   echo "       $0 psql.conf diku";
+   
+   exit 1;
+fi
+
+DBCONF=${1}
+export TENANT=${2}
+
+export PGDATABASE=`cat $DBCONF | jq '.database' | cut -d\" -f2`
+export PGUSER=`cat $DBCONF | jq '.username' | cut -d\" -f2`
+export PGPASSWORD=`cat $DBCONF | jq '.password' | cut -d\" -f2`
+export PGHOST=`cat $DBCONF | jq '.host' | cut -d\" -f2`
+export PGPORT=`cat $DBCONF | jq '.port' | cut -d\" -f2`
+
+LOANS="loans.tsv"
+REQUESTS="requests.tsv"
+# AUDIT_LOANS="audit_loans.tsv"

```

Comment:
Note that I commented out the step to populate audit_loans.tsv. This is because there are PK conflicts, which I can't figure out why. Audit_loans table is nice that important to have populated, but not a must. Will circle back to it soon.